### PR TITLE
fixed bug with rejected and new-submition empty link

### PR DESCRIPTION
**Pull request by borysyuk**

### CODE REVIEW

File: census/models/utils.js [hunk 1]
```diff
@@ -133,13 +133,17 @@ var processEntries = function(data, options) {
     data.submitters = [];
 
     if (options.cascade) { data.entries = cascadeEntries(data.entries, options.year); }
+
+    _.each(data.entries, function(e) {
+      e.computedYCount = e.yCount(data.questions);
```
Comment (pwalsh) [inline census/models/utils.js]: @borysyuk why do we also need to move the computedYCount?

Comment (borysyuk) [inline census/models/utils.js]: @pwalsh We divide entries on several groups (pending, rejected and other). So If we need this property in all entries we should initialize this property before dividing.

Comment (pwalsh) [inline census/models/utils.js]: Yes, I know. Can you check though? I think we only need it on entries, and therefore calculating it for all variants is a waste (but, I could be wrong, so please confirm)

Comment (borysyuk) [inline census/models/utils.js]: We use computedYCount property for "entries" and "pending" group and we use this property in overview.html in class attribute too  (class="ycount-xxx"  xxx - value of property). But i can not found where this class is using. Seems like we don't need calculate it for rejected.

**Pull request merged**